docs/architecture/README.md

```diff
@@ -78,8 +78,9 @@ When writing ADRs, follow the same best practices for writing RFCs. When writing
 * [ADR 039: Epoched Staking](./adr-039-epoched-staking.md)
 * [ADR 040: Storage and SMT State Commitments](./adr-040-storage-and-smt-state-commitments.md)
 * [ADR 046: Module Params](./adr-046-module-params.md)
+* [ADR 050: Auto-Compound Rewards](./adr-050-auto-compound-rewards.md)
```

Comment:
can we use another number? i think we already talked a couple of times about adr-050 referring to sign-mode-textual